pkgs/development/python-modules/pykka/default.nix

```diff
@@ -1,27 +1,20 @@
-{ stdenv
-, buildPythonPackage
-, fetchFromGitHub
-}:
+{ lib, buildPythonPackage, fetchPypi, pytestCheckHook, pytest-mock }:
 
 buildPythonPackage rec {
-  pname = "pykka";
-  version = "2.0.1";
+  pname = "Pykka";
+  version = "2.0.3";
 
-  src = fetchFromGitHub {
-    owner = "jodal";
-    repo = pname;
-    rev = "refs/tags/v${version}";
-    sha256 = "011rvv3vzj9rpwaq6vfpz9hfwm6gx1jmad4iri6z12g8nnlpydhs";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4b9d2363365b3455a0204bf163f09bd351d24b938c618c79d975a9510e128e95";
   };
 
-  # There are no tests
-  doCheck = false;
+  checkInputs = [ pytestCheckHook pytest-mock ];
 
-  meta = with stdenv.lib; {
-    homepage = "http://www.pykka.org";
+  meta = with lib; {
+    homepage = "https://www.pykka.org/";
     description = "A Python implementation of the actor model";
+    changelog = "https://github.com/jodal/pykka/blob/v${version}/docs/changes.rst";
     license = licenses.asl20;
-    maintainers = [];
```

Comment:
We should probably add one maintainer here.

Reply:
or remove the package

Reply:
I added myself as maintainer :)